setup.py

```diff
@@ -23,7 +22,7 @@
         'pyvistaqt'
     ],
     url='https://github.com/cgre-aachen/gempy',
-    download_url='https://github.com/cgre-aachen/gempy/archive/2.1.1tar.gz',
+    download_url='https://github.com/cgre-aachen/gempy/archive/2.2.1.tar.gz',
```

Comment:
I would remove download_url altogether. This was always, I think, just additional meta-data, and I don't think it is used at all. See, e.g. https://github.com/pypa/setuptools_scm/issues/95#issuecomment-239774633

Reply:
done!